src/components/modules/NavSidebar/NavSidebar.view.vue

```diff
@@ -4,7 +4,7 @@
       <li>
         <router-link
           v-slot="{ isActive, navigate }"
-          :to="{ name: 'Oauth' }"
+          :to="{ name: 'Oauth', query: queryWithClientId }"
```

Comment:
why not just `query`?

Reply:
I guess it's good to make it explicit, because if will only query, it always will root
here we have link to Oauth route, for now it same with root, but if we change Oauth path, it will be different

Reply:
query have other props and the should be passed too
for example `openMode`
please, rename it to single `query`